websites/S/Steam/presence.ts

```diff
@@ -23,314 +19,471 @@ presence.on("UpdateData", async () => {
 
 		if (document.location.pathname === "/" || !document.location.pathname) {
 			if (subsection) {
-				presenceData.state = `Browsing ${subsection}.`;
-
+				presenceData.state = `Browsing ${subsection}`;
 				presenceData.startTimestamp = browsingTimestamp;
 			} else {
-				presenceData.state = "Home";
-
+				presenceData.state = "Homepage";
 				presenceData.startTimestamp = browsingTimestamp;
 			}
+		} else if (document.location.pathname.includes("/filedetails/")) {
+			presenceData.details = "Viewing a community post";
+			presenceData.state = document
+				.querySelector<HTMLTitleElement>("head > title")
+				.textContent.replace("Steam Community :: ", " ")
+				.replace("::", "|");
+			presenceData.startTimestamp = browsingTimestamp;
+			presenceData.buttons = [
+				{
+					label: "View the page",
+					url: document.location.href,
+				},
+			];
+		} else if (
+			document.location.pathname.includes("/id") ||
+			document.location.pathname.includes("/profiles")
+		) {
+			if (
+				document.location.pathname.includes("/badges") ||
+				document.location.pathname.includes("/games") ||
+				document.location.pathname.includes("/inventory") ||
+				document.location.pathname.includes("/recommended") ||
+				document.location.pathname.includes("/gamecards") ||
+				document.location.pathname.includes("/awards")
+			) {
+				pfname = document.querySelector(
+					"#responsive_page_template_content > div > div.profile_small_header_bg > div > div > span.profile_small_header_name > a"
+				).innerText;
```

Comment:
Use `textContent` instead, also trim the content.

![image](https://user-images.githubusercontent.com/68460474/209691338-7d343706-2994-42ef-b43d-7e0f6bed3769.png)


```suggestion
				)?.textContent.trim();
```

Reply:
Thank you for the suggestion👌